src/shared/components/CloudUpgradeButton.tsx

```diff
@@ -47,28 +50,43 @@ const CloudUpgradeButton: FC<OwnProps> = ({
     history.push('/checkout')
   }
 
-  const promoMessage =
-    isFlagEnabled('credit250Experiment') && showPromoMessage ? (
-      <span className="credit-250-experiment-upgrade-button--text">
-        Get $250 free credit
-      </span>
-    ) : null
+  const original = showUpgradeButton ? (
+    <Button
+      className={cloudUpgradeButtonClass}
+      icon={IconFont.CrownSolid_New}
+      onClick={handleClick}
+      shape={ButtonShape.Default}
+      size={size}
+      testID="cloud-upgrade--button"
+      text={buttonText}
+    />
+  ) : null
```

Comment:
if we're going to return `null` from an if statement in the context of a `render` function, this can just be shortened to:

```ts
conditional && (
  returnValue
)
```

to avoid an empty `else` block. `null`, `undefined`, and boolean values are all equivalent when it comes to rendering JSX. https://stackoverflow.com/a/42083262/92446